entrypoint.sh

```diff
@@ -2,5 +2,6 @@
 
 # Guarantee that we are on the same path as our *.js scripts
 cd /
-npm ci  # more suitable for installs from scratch 
-npm start
+npm uninstall puppeteer
```

Comment:
Can you explain why the need to ` npm uninstall puppeteer` and then `npm install puppeteer`?

Reply:
please see https://github.com/Schrodinger-Hat/youtube-to-anchorfm/pull/78#issuecomment-1449972866